doc/crypto/X509_new.pod

```diff
@@ -46,7 +46,7 @@ If the allocation fails, X509_new() returns B<NULL> and sets an error
 code that can be obtained by L<ERR_get_error(3)>.
 Otherwise it returns a pointer to the newly allocated structure.
 
-X509_free() and X509_up_ref() do not return a value.
+X509_free() do not return a value.
```

Comment:
"does not return"  but is that even needed for void functions?  Let's just remove those lines.